docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple Check
+====================
+
+Description
+-----------
+
+The SimpleCheck gives you basic information about your datasets and detects the most obvious of datashifts.
+
+Some shifts result in the change of basic metrics, that are commonly used for the analysis and exploration of datasets.
+Those metrics and statistics are calculated on both datasets and compared afterwards. HIER NOCH AMAZON DEEQU ERWÄHNEN,
+WIRD NACH UMBENENNUNG DES CHECKS GEMACHT
```

Comment:
Create an Issue for that and delete the comment from the documentation

Reply:
Good idea. [#125](https://github.com/hpi-bp1819-naumann/shift-detector/issues/125)